source/alloc/Cargo.toml

```diff
@@ -24,3 +24,7 @@ features = ["defmt-impl"]
 [dependencies.linked_list_allocator]
 version = "0.10.1"
 default-features = false
+
+[features]
+default = []
+use-std = []
```

Comment:
nit, tioli: i feel like most of our other crates just call this feature flag "std" rather than "use-std"...

Reply:
```
➜  mnemos git:(james/wtf-alloc) rg -i '("use-std"|use-std =)'
tools/dumbloader/Cargo.toml
10:features = ["use-std"]

tools/f3repl/Cargo.toml
12:features = ["use-std"]

source/melpomene/Cargo.toml
33:features = ["use-std"]

source/forth3/Cargo.toml
15:use-std = []

source/forth3/src/lib.rs
3:#![cfg_attr(not(any(test, doctest, feature = "use-std")), no_std)]
14:#[cfg(any(test, doctest, feature = "use-std"))]

source/forth3/src/vm/builtins/floats.rs
66:    #[cfg(feature = "use-std")]
74:    #[cfg(not(feature = "use-std"))]

source/alloc/Cargo.toml
30:use-std = []

source/alloc/src/lib.rs
1:#![cfg_attr(not(feature = "use-std"), no_std)]

source/alloc/src/heap.rs
78:#[cfg(feature = "use-std")]
```

```
$ rg -i '("std"|std =)'

source/melpomene/Cargo.toml
89:features = ["std", "async-await"]

source/trace-proto/Cargo.toml
7:std = ["tracing-serde-structured/std", "serde/std"]

source/trace-proto/src/lib.rs
1:#![cfg_attr(not(feature = "std"), no_std)]

tools/crowtty/Cargo.toml
37:features = ["std"]
```

I think this boils down to "eliza uses "std", james uses "use-std"

Reply:
lol, welp. maybe we should make this consistent...